README.md

```diff
@@ -1,3 +1,20 @@
+### libsass version support
+
+The libsass library has several API changes over time.  So, the code has dependencies on the libsass version.
+Look at the file sass/version.h to find your installed libsass version.  This code should work with 3.3.x, 3.4.x and when released 3.5.x.
```

Comment:
I think this is correct.
I have tested 3.3.4 using apt-get install libsass-dev in Ubuntu 16.04 in Docker.  I have tested 3.4.x from a source build to match FreeBSD current version.   At this time the 3.5 should work if you define HAVE_SASS_DELETE_OPTIONS.